.changeset/brave-teachers-burn.md

```diff
@@ -0,0 +1,5 @@
+---
+"saleor-dashboard": minor
```

Comment:
suggetion: IMO this change doesn't require change in snapshot, so we could mark it as patch and run on 3.20